spec/swagger_helper.rb

```diff
@@ -117,11 +117,64 @@
           properties: {
```

Comment:
This ought to always match the parameters returned by the associated serializer. I wonder if we could add an automated test that ensures they match for all API resources.